ui/components/app/incoming-trasaction-toggle/network-toggle.tsx

```diff
@@ -68,21 +70,39 @@ const NetworkToggle = ({
           src={networkPreferences.imageUrl}
           name={networkName}
         />
-        <Text
-          color={TextColor.textDefault}
-          backgroundColor={BackgroundColor.transparent}
-          variant={TextVariant.bodyMd}
-          ellipsis
-          marginLeft={2}
-        >
-          {networkName.length > MAXIMUM_CHARACTERS_WITHOUT_TOOLTIP ? (
-            <Tooltip title={networkName} position="bottom">
-              {networkName}
-            </Tooltip>
-          ) : (
-            networkName
-          )}
-        </Text>
+        <Box display={Display.Flex} flexDirection={FlexDirection.Column}>
+          <Text
+            color={TextColor.textDefault}
+            backgroundColor={BackgroundColor.transparent}
+            variant={TextVariant.bodyMd}
+            ellipsis
+            marginLeft={2}
+          >
+            {networkName.length > MAXIMUM_CHARACTERS_WITHOUT_TOOLTIP ? (
+              <Tooltip title={networkName} position="bottom">
+                {networkName}
+              </Tooltip>
+            ) : (
+              networkName
+            )}
+          </Text>
+          <Text
+            color={TextColor.primaryDefault}
+            backgroundColor={BackgroundColor.transparent}
+            variant={TextVariant.bodySm}
+            ellipsis
+            marginLeft={2}
+          >
+            <a
+              key={`network_${domain}_link`}
+              href={`https://${domain}`}
+              rel="noreferrer"
+              target="_blank"
+            >
+              {domain.charAt(0).toUpperCase() + domain.slice(1)}
```

Comment:
Optional: might be more clear to assign to a variable to have this transform explained and easier to read.